tests/test_transforms.py

```diff
@@ -88,6 +88,37 @@ def test_normal(self):
     pass
 
 
+class TestSubstituteWords(unittest.TestCase):
+    def test_normal(self):
+        cases = [
+            (["you're pretty"], ["i am awesome"]),
+            (["your book"], ["your book"]),
+            (["foobar"], ["foobar"]),
+        ]
+
+        _apply_test_on(
+            self,
+            SubstituteWords(
+                {"pretty": "awesome", "you": "i", "'re": " am", "foo": "bar"}
+            ),
+            cases,
+        )
+
+
+class TestApplyRegexSubstitution(unittest.TestCase):
+    def test_normal(self):
+        cases = [
+            (["is the world doomed or loved?"], ["is the world sacr or lov?"]),
+            (["the sun is loved"], ["the sun is lov"]),
+        ]
```

Comment:
You could add something like `(["edibles are allegedly cultivated"], ["edibles are allegedly cultivat"])` to show that "ed" is only replaced at the end of words